src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -18,6 +18,8 @@ class HookPhabricator(Hook):
     Taskcluster hook handling the static analysis
     for Phabricator differentials
     '''
+    top_id = None
```

Comment:
Maybe rename it as `latest_id` to make it clearer that it is the latest ID we have analyzed

Reply:
way better name, thanks